### PR TITLE
Transform com_finder to ES6 and remove inline script  [ Part1 ]

### PR DESCRIPTION
**Pull request by anuragteapot**
Pull Request for Issue 
1.   #8 
2.  #55

### CODE REVIEW

**Comment by anuragteapot** [comment]
We can't make dog happy  :(

**Review by C-Lodder — APPROVED** [review]

**Comment by C-Lodder** [comment]
@Anu1601CS dont worry about the `no-alert` warnings

**Review by dneukirchen — CHANGES_REQUESTED** [review]
Rest looks fine. Thank you.

File: media/com_finder/js/filters.es6.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Joomla = window.Joomla || {};
+
+(() => {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', () => {
+    Joomla.submitbutton = (pressbutton) => {
+      if (pressbutton === 'filters.delete') {
+        if (window.confirm(Joomla.JText._('COM_FINDER_INDEX_CONFIRM_DELETE_PROMPT'))) {
```
Comment (dneukirchen) [inline media/com_finder/js/filters.es6.js]: We can simplify this and remove 1 if and the else clause.

```
if (pressbutton === 'filters.delete' && !window.confirm(Joomla.JText._('COM_FINDER_INDEX_CONFIRM_DELETE_PROMPT'))) {
    return false 
}
...
```

**Review by dneukirchen — APPROVED** [review]

**Pull request merged**